src/Symfony/Bridge/Twig/Tests/AppVariableTest.php

```diff
@@ -47,9 +47,8 @@ public function testEnvironment()
 
     public function testGetSession()
     {
-        $session = $this->getMockBuilder(Session::class)->disableOriginalConstructor()->getMock();
```

Comment:
why is this changed again?

Reply:
Because of the fact that the session is constructed using an instance of `NativeSessionStorage` that by default used the deprecated `SessionHandlerProxy`, this test did not pass due to the deprecation error. Now there are no deprecations, so no need for disabling the original constructor,

Reply:
We should be using `MockArraySessionStorage` rather than `NativeSessionStorage` in tests (or create a mock as before).